tests/test_pattern_matching.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from returns.io import IO, IOFailure, IOResult, IOSuccess
+from returns.maybe import Maybe, Nothing, Some
+from returns.result import Failure, Result, Success
+
+
+@pytest.mark.parametrize('container', [
```

Comment:
Let's use `skipif` here, it would be more readable! 👍

Reply:
Instead on custom `pytest` hook.

Reply:
I think I just need to fix this one here, I'll try to do it tomorrow

Reply:
I've add a marker `min_python_3_10`, I think it's more readable than `skipif`

Reply:
I've added a marker `min_python_3_10`, I think it's more readable than `skipif`

Reply:
Well, it doesn't work. `skipif` neither

Reply:
Python evaluates the function body

Reply:
As you can see here: https://github.com/dry-python/returns/pull/991/checks?check_run_id=3563951538

We had a problem in the collect part

Reply:
I've rolled back!